gwctl/pkg/printer/gatewayclasses_test.go

```diff
@@ -250,3 +252,175 @@ Status: {}
 		})
 	}
 }
+
+// TestGatewayClassesPrinter_PrintJson tests the -o json output of the `get` subcommand
+func TestGatewayClassesPrinter_PrintJson(t *testing.T) {
```

Comment:
Can we please squash the separate tests for PrintJSON and PrintYAML into one, since they _should_ mostly be the same with the same setup, with the only difference in the assertion/comparision of output.

Reply:
Sure, sorry for leaving them like this, they were a consequence of the previous design of separate JSON and YAML printers.